kube_apiserver_metrics/README.md

```diff
@@ -26,7 +26,7 @@ annotations:
     '[{ "prometheus_url": "https://%%host%%:%%port%%/metrics", "bearer_token_auth": "true" }]'
 ```
 
-Then the Datadog Cluster Agent schedules the check(s) for each endpoint onto Datadog Agent(s).
+Then the Datadog Cluster Agent schedules the check(s) for each endpoint onto Datadog Agent(s). Recall that you must add `cluster_check:true` to your configuration file when using a ConfigMap to configure cluster checks.
```

Comment:
should we make it more specific and link the configuration file?

Reply:
`+`

Reply:
agree that we should link out